src/components/Firestore/variables.scss

```diff
@@ -15,7 +15,7 @@
  */
 
 // firestore card
-$firestore-card-min-height: 400px;
+$firestore-card-min-height: 600px;
```

Comment:
When the card height is `400px`, it is difficult to scroll because the SubCollection area is too small. `600px` is much better. It is about the same as in the Firebase Console.

## 400px
![400px](https://user-images.githubusercontent.com/2603644/156474318-72b35edf-90c7-4c29-8503-6a4cc927168c.gif)

## 600px
![600px](https://user-images.githubusercontent.com/2603644/156474321-e663cf9a-5d11-4eb0-b5d4-79592f967a12.gif)

## Firebase Console
<img width="865" alt="スクリーンショット 2022-03-03 9 59 22" src="https://user-images.githubusercontent.com/2603644/156475713-181a6fd3-9978-412c-8314-e0090541fe6d.png">